src/test/CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ add_executable(${BINARY}
     ../utils/FileUtils.cpp
     ../utils/JsonUtils.cpp
     ../utils/StringUtils.cpp
+    ../utils/ThreadPool.cpp
```

Comment:
when removed from AdaptiveTree (my previous comment) this addition should not be needed